packages/@okta/vuepress-site/docs/reference/api-overview/index.md

```diff
@@ -121,6 +121,8 @@ The `Accept-Language` HTTP header advertises which languages the client is able
 
 The `X-Device-Fingerprint` HTTP header supplies the device fingerprint used in an authentication request.
 
+> **Note:** test
```

Comment:
we could also add a note here, but not sure it is needed since there is no mention of behavior detection in this page